modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotMetadata.java

```diff
@@ -137,7 +137,7 @@ public int pageSize() {
     }
 
     /**
-     * @return The list of cache groups ids which were included into snapshot.
+     * @return The list of cache groups ids which were included into the snapshot globally.
```

Comment:
"cache groups ids" can be changed to "cache group IDs"
it's up to you, but I found the expression "cache group ids" only in snapshots

Reply:
Fixed.